client/components/ManageTestQueue/index.jsx

```diff
@@ -141,12 +195,360 @@ const ManageTestQueue = ({
+                                          row.browser.id ===
+                                              updateBrowserIdForUpdate &&
+                                          row.phase == updatePhaseForUpdate
+                                      ) {
+                                          return false;
+                                      }
+                                      return true;
+                                  })
+                                  .sort((a, b) => {
+                                      if (a.phase < b.phase) return -1;
+                                      if (a.phase > b.phase) return 1;
+                                      return a.at.name.localeCompare(b.at.name);
+                                  })
+                          );
+                      }
+                  } catch (error) {
+                      setShowThemedModal(true);
+                      setThemedModalTitle(
+                          'Error Updating Required Reports Table'
+                      );
+                      setThemedModalContent(<>{error.message}</>);
+                  }
+              }, 'Adding Phase requirement to the required reports table')
+            : mutation === 'deleteRequiredReport'
+            ? await triggerLoad(async () => {
+                  const { data } = await deleteRequiredReport({
+                      variables: {
+                          atId: updateAtIdForUpdate,
+                          browserId: updateBrowserIdForUpdate,
+                          phase: `IS_${updatePhaseForUpdate}`
+                      }
+                  });
+
+                  const deletedRequiredReport =
+                      data.requiredReport.deleteRequiredReport;
+
+                  if (deletedRequiredReport) {
+                      setAtBrowserCombinations(
+                          [...atBrowserCombinations]
+                              .filter(row => {
+                                  if (
+                                      row.at.id === updateAtIdForUpdate &&
+                                      row.browser.id ===
+                                          updateBrowserIdForUpdate &&
+                                      row.phase == updatePhaseForUpdate
+                                  ) {
+                                      return false;
+                                  }
+                                  return true;
+                              })
+                              .sort((a, b) => {
+                                  if (a.phase < b.phase) return -1;
+                                  if (a.phase > b.phase) return 1;
+                                  return a.at.name.localeCompare(b.at.name);
+                              })
+                      );
+                  }
+              }, 'Adding Phase requirement to the required reports table')
+            : null;
```

Comment:
I'm not categorically against `a ? b : c ? d : e` type of code but I do think the way the if statements are spread across so many lines I think it would be more readable to use an `if else {}` syntax.